schema/schema.sql

```diff
@@ -0,0 +1,64 @@
+--------- Create book table ----------------
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  publisher VARCHAR,
+  cover_state VARCHAR,
+  genre_id INT NULL REFERENCES genres(id), -- To be created later
+  author_id INT NULL REFERENCES authors(id),-- To be created later
+  label_id INT NULL REFERENCES labels(id),-- To be created later
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL
+);
+
+----------- Create labels table ------------------
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(50) NOT NULL,
+  color VARCHAR(50) NOT NULL
+);
+
+------------ Create music album table ----------------
+CREATE TABLE music_albums (
+  id SERIAL PRIMARY KEY,
+  publish_date DATE NOT NULL DEFAULT DATE,
+  on_spotify BOOLEAN,
+  archived BOOLEAN,
+  genre_id INT,
+  label_id INT,
+  author_id INT,
+  CONSTRAINT fk_genre FOREIGN KEY (genre_id) REFERENCES genres(genre_id)
+  CONSTRAINT fk_label FOREIGN KEY (label_id) REFERENCES label(label_id)
+  CONSTRAINT fk_author FOREIGN KEY (author_id) REFERENCES authors(author_id)
+  PRIMARY KEY(id)
+);
+
+------------ Create genre table -------------------
+CREATE TABLE genres (
+  genre_id SERIAL PRIMARY KEY,
+  name VARCHAR(100),
+  PRIMARY KEY(genre_id)
+);
+
+--------------- Create game table ------------------
+CREATE TABLE game (
+  id SERIAL PRIMARY KEY,
+  multiplayer BOOLEAN,
+  last_played_at DATE,
+  genre_id INT,
+  label_id INT,
+  publish_date DATE,
+  author_id INT,
+  CONSTRAINT fk_genre FOREIGN KEY (genre_id) REFERENCES genres(genre_id)
+  CONSTRAINT fk_label FOREIGN KEY (label_id) REFERENCES label(label_id)
+  CONSTRAINT fk_author FOREIGN KEY (author_id) REFERENCES authors(author_id)
+  PRIMARY KEY(id)
+);
+
+-------------- Create author table -------------------
+CREATE TABLE author (
+    id SERIAL PRIMARY KEY,
+    first_name  VARCHAR(100),
+    last_name   VARCHAR(100)
+);
+
```

Comment:
- [ ] Before creating the tables, it is intuitive to have the database itself created. KIndly include the line for creating the database as well at the top of the file.